docs/tutorials/edl.ipynb

```diff
@@ -0,0 +1,98 @@
+{
```

Comment:
This is a really helpful notebook for testing. Will this get exposed in the documentation? If so, we may want to add more explanation here (how this differs from authenticate.md for example.

Reply:
I was thinking of adding it but maybe all these examples should be in a gallery project so they are not tied to releases.